pages/bindingDescription.xml

```diff
@@ -71,6 +71,142 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <p>You can always further specify this information within one
         of the <gi>decoNote</gi> elements within
         <gi>binding</gi>.</p>
-        </body>
+
+    <div type="level3">
+      <head>Boards</head>
+        <p>In the
+            <tag>decoNote type="Boards"</tag>
+            you can specify using keywords the wood type (if known) and
+            encode the presence of a textile inlay or of a mirror.
+        </p>
+    </div>
+
+    <div type="level3">
+      <head>Cover</head>
+       <p>In the
+             <tag>decoNote type="Cover"</tag>, using the keywords,
```

Comment:
please add a link to relevant page on keywords http://betamasaheft.eu/Guidelines/?q=term&id=keywordsIntro

Reply:
see my commit